src/test/java/Homework22.java

```diff
@@ -0,0 +1,35 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+public class Homework22 extends BaseTest {
+    @Test
+    public void deletePlaylist() throws InterruptedException {
+
+        String expectedPlaylistDeletedMessage = "Deleted playlist \"Homework17.\"";
+
+        navigateToPage();
+        provideEmail("john.wolschleger@testpro.io");
+        providePassword("te$t$tudentb49");
+        clickSubmit();
+        clickPlaylistsGrid();
+        clickDeletePlaylistBtn("Homework17");
+        Assert.assertEquals(getDeletePlaylistSuccessMsg(), expectedPlaylistDeletedMessage);
+    }
+    public void clickPlaylistsGrid() throws InterruptedException {
+        WebElement viewAll = wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//a[contains(text(), \"Homework17\")]")));
+        viewAll.click();
+    }
+    public void clickDeletePlaylistBtn(String name) throws InterruptedException {
+        WebElement playlistBtn = wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//button[@class='del btn-delete-playlist']")));
+        //WebElement playlistBtn = driver.findElement(By.xpath("//button[@class='del btn-delete-playlist']"));
+        playlistBtn.click();
+        //Thread.sleep(2000);
+    }
+    public String getDeletePlaylistSuccessMsg() {
+        WebElement notification = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("div.success.show")));
+        //WebElement notification = driver.findElement(By.cssSelector("div.success.show"));
+        return notification.getText();
+    }
```

Comment:
Move these methods to the corresponding class according to the POM.

For example

![image](https://github.com/Test-Pro-Bootcamp/A53-QA-Automation-Framework/assets/104761482/82e22e49-d71f-43c9-b796-6f0f4d72799a)


![Clean_Test](https://github.com/Test-Pro-Bootcamp/A53-QA-Automation-Framework/assets/104761482/04fe330d-bc22-435d-ba32-cb55f208f91c)